### PR TITLE
Added Niall Kehoe

### PR DESCRIPTION
**Pull request by niallkehoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,10 @@ To add your own project below, just [edit](https://github.com/wwdc/2020/edit/mas
 |[Aryan Nambiar](https://twitter.com/ifisq)|[Github](https://github.com/ifisq/Build-A-Piano)||UIKit, AVFoundation|Submitted|
 |[Aurther Nadeem](https://www.instagram.com/aurther_nadeem/)|[GitHub](https://github.com/Aurther-Nadeem/WWDC2020)||ARKit, RealityKit, SwiftUI, AVFoundation|Submitted|
 |[Benjamin Hutter](https://www.instagram.com/benjaminhtr/)|[GitHub](https://github.com/benjaminhtr/WWDC20)||UIKit, CoreML, Vision, AVFoundation|Submitted|
-|[Berkin Ceylan](https://twitter.com/BerkinCeylan)|[Github](https://github.com/berkinceylan/WWDC20)||SwiftUI|Submitted|
```
Comment (zmknox) [inline README.md:22]: Make sure you aren't removing entries. I've been actively merging things, so conflicts are likely

File: README.md [hunk 2]
```diff
@@ -19,12 +19,10 @@ To add your own project below, just [edit](https://github.com/wwdc/2020/edit/mas
...
 |[Christian P](https://twitter.com/Priva2804)|[Github](https://github.com/Priva28/PlanetARium)|[Youtube](https://youtu.be/EwPspV8ZUp4)|SwiftUI, ARKit, SceneKit, Vision|Submitted|
 |[Deniz Karakay](https://www.instagram.com/sedna_karakay)|[Github](https://github.com/dkarakay/Stop-Pandemic)||SpriteKit, AVFoundation, SwiftUI|Submitted|
 |[Hengyu Liu](https://www.instagram.com/a211212abc/)|[GitHub](https://github.com/a211212abc/WWDC20)||SwiftUI, ARKit, SpriteKit|Submitted|
 |[Henrik Storch](https://twitter.com/thisisthefoxe)|[GitHub](https://github.com/thisIsTheFoxe/WWDC20)|(later)|SpriteKit|Submitted|
-|[Ihwan D](https://ihwan.id)|[GitHub](https://github.com/IhwanID/wwdc20-rice-cooker-hack)|[YouTube](https://youtu.be/0fgdYEAn6MQ)|SwiftUI, AVFoundation|Submitted|
```
Comment (zmknox) [inline README.md:27]: here too

**Review by zmknox — CHANGES_REQUESTED** [review]

**Comment by zmknox** [comment]
Also @nkehoegames You don't need to create a new PR for every change. You can commit more to your branch and they will be added to your existing PR

**Comment by niallkehoe** [comment]
Sorry, I clicked edit and it must not have updated it properly. Is it okay or do I need to edit it?

**Comment by zmknox** [comment]
Your first PR was fine, I just couldn't resolve conflicts on it because you set your fork such that I cannot push to it. If you can enable that, I can fix it. https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by niallkehoe** [comment]
It seems like its already enabled.

**Comment by zmknox** [comment]
Closing as duplicate

**Pull request closed without merging**